### DISCUSSION TITLE
FormPath 相对路径在 schema 模式中没有生效

### DISCUSSION THREAD

**Discussion by TanYJie** [alibaba/formily#1516] [Q&A 2.x] [answered]
在 formily 的文档中，我看到可以使用点运算符来实现相对路径的查找：https://core.formilyjs.org/api/entry/form-path#%E7%9B%B8%E5%AF%B9%E8%B7%AF%E5%BE%84

但是在 schemaForm 的联动协议中使用点运算符，没有实现我想达到的联动效果。
我没有搜索到相关的 discussion，因此提出这个问题。

SchemaForm sandbox：https://codesandbox.io/s/20-test-iwvnd?file=/App.tsx

**Comment by janryWang** [comment]
taget不支持传相对路径，如果希望走相对路径模式，请使用dependencies

**Reply by wilsoncook** [reply]
不支持/语法不同的话，文档应该说明一下，一个黑盒逻辑会让挺多人浪费太多时间

**Comment by liuweiGL** [comment, marked as answer]
https://codesandbox.io/s/20-test-forked-iio6u?file=/App.tsx  你看看我这个是没问题的，你那个示例控制台都报错了，导致代码执行异常。

**Reply by elsonz** [reply]
https://codesandbox.io/s/20-test-forked-xye71?file=/App.tsx
即使解决了 FormLayout 的报错也不行。
1. 只要目标字段，即 demo 里的 input 字段不在最顶层，就不会生效。
2. 另外 select 字段并没有控制同级的 input 字段，而是控制了最顶层的 input

**Comment by liuweiGL** [comment]
https://codesandbox.io/s/20-test-forked-2shpk?file=/App.tsx
